v4l2loopback.c

```diff
@@ -73,26 +73,27 @@ MODULE_LICENSE("GPL");
 #define dprintk(fmt, args...)                                                  \
 	do {                                                                   \
 		if (debug > 0) {                                               \
-			printk(KERN_INFO "v4l2-loopback[" STRINGIFY2(          \
+			printk(KERN_INFO "pid(%d): v4l2-loopback[" STRINGIFY2( \
 				       __LINE__) "]: " fmt,                    \
-			       ##args);                                        \
+			       task_pid_nr(current), ##args);                  \
 		}                                                              \
 	} while (0)
 
-#define MARK()                                                                 \
-	do {                                                                   \
-		if (debug > 1) {                                               \
-			printk(KERN_INFO "%s:%d[%s]\n", __FILE__, __LINE__,    \
-			       __func__);                                      \
-		}                                                              \
+#define MARK()                                                           \
+	do {                                                             \
+		if (debug > 1) {                                         \
+			printk(KERN_INFO "pid(%d): %s:%d[%s]\n",         \
+			       task_pid_nr(current), __FILE__, __LINE__, \
+			       __func__);                                \
+		}                                                        \
```

Comment:
```suggestion
#define MARK()                                                                 \
	do {                                                                   \
		if (debug > 1) {                                               \
			printk(KERN_INFO "pid(%d): %s:%d[%s]\n",               \
			       task_pid_nr(current), __FILE__, __LINE__,       \
			       __func__);                                      \
		}                                                              \
```

Avoid re-aligning lines unnecessarily; reduces the overall patch to just the modified lines inside that `#define`.

Reply:
Done